<a href="https://colab.research.google.com/github/sebastiancasas10/curso-ia-para-economia/blob/main/Taller_Limpieza_Transformacion_Datos_Sebastian_Casas.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# **Inteligencia Artificial con Aplicaciones en Economía I**

- 👩‍🏫 **Profesora:** [Lina María Castro](https://www.linkedin.com/in/lina-maria-castro)  
- 📧 **Email:** [lmcastroco@gmail.com](mailto:lmcastroco@gmail.com)  
- 🎓 **Universidad:** Universidad Externado de Colombia - Facultad de Economía

# **Taller Limpieza y transformación de datos**🧹

**IMPORTANTE**: Guarda una copia de este notebook en tu Google Drive o computador.

**Nombres estudiantes:**

- Sebastián Casas Poloche


**Forma de entrega:**

- Nombrar el archivo de la siguiente forma:“Taller_Limpieza_nombres_y_appellidos.ipynb”.
- Suba el Jupyter Notebook a su cuenta en Github y envíe el link en el siguiente Forms: https://forms.office.com/r/6GZ89uiLNC.

**IMPORTANTE:** No se recibirán talleres en Google Colab, el notebook debe estar subido en Github.

**Plazo de entrega:**

19 de marzo de 2026, máximo a las 11:59 p.m. Tenga en cuenta que luego de esa hora el formulario en forms se cierra. El Jupupyter Notebook también debe quedar subido en Github antes de esa hora.

**Instrucciones Generales:**

Completa el código en las celdas marcadas con `### TU CÓDIGO AQUÍ ###`. Puedes añadir más celdas si lo requieres.

Utilice la base de la GEIH de julio a diciembre de 2025, la cual se encuentra en https://drive.google.com/drive/folders/1Th4TWdsOahAAvBUKxixFhaELC_KPsl3y?usp=sharing. Tenga en cuenta sep="," y encoding='latin1'. Las variables 'DPTO' y 'RAMA4D_R4' deben ser cargadas como string.

# Ejercicio 1: Importar librerías y cargar datos

1.	Importe todas las librerías que vaya a utilizar.

In [47]:
import os
import numpy as np
import pandas as pd
import warnings
warnings.filterwarnings('ignore')

2.	Mejore la visualización de los dataframes como se ha visto en clase.

In [48]:
# Que muestre todas las columnas
pd.options.display.max_columns = None
# En los dataframes, mostrar los float con dos decimales
pd.options.display.float_format = '{:,.2f}'.format

3.	Conéctese a su Google Drive y establezca la ruta del dataset.

In [49]:
from google.colab import drive, files
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


4.	Cargue los datos en el notebook. Llame al dataframe 'df_geih'

In [50]:
path = '/content/drive/MyDrive/dataset_taller_5'
os.chdir(path)

In [51]:
df_geih = pd.read_csv('GEIH_2025_Jul_Dic_Consolidado.csv', sep=",", encoding='latin1', dtype={'DPTO': str, 'RAMA4D_R4': str})


5. Obtenga las dimensiones de la base

In [52]:
df_geih.shape

(409127, 314)

# Ejercicio 2:  Seleccionar y renombrar columnas

La GEIH contiene cientos de variables. Para nuestro análisis del mercado laboral, nos enfocaremos en un subconjunto.

Adicionalmente, los nombres de las columnas en la GEIH son códigos. Para hacer el análisis más intuitivo, vamos a renombrarlas.

1. Crea un nuevo DataFrame llamado `df_empleo` que contenga únicamente las columnas que están en la lista `columnas_interes`.

In [53]:
columnas_interes = ['DIRECTORIO', 'SECUENCIA_P', 'ORDEN', 'MES', 'CLASE', 'DPTO', 'FEX_C18', 'PT', 'P3271', 'P6040', 'P6080', 'P3043', 'OCI',
                    'P6430', 'RAMA4D_R4', 'INGLABO', 'DESCRIPCION_CIIU', 'NOMBRE_DPTO']

In [54]:
df_empleo = df_geih[columnas_interes]
df_empleo.head()


,DIRECTORIO,SECUENCIA_P,ORDEN,MES,CLASE,DPTO,FEX_C18,PT,P3271,P6040,P6080,P3043,OCI,P6430,RAMA4D_R4,INGLABO,DESCRIPCION_CIIU,NOMBRE_DPTO
0,8283143,1,1,7,2,08,27.75,1,1,72,6,NaN,1.00,7.00,0143,"790,000.00",Cría de ovejas y cabras,ATLÁNTICO
1,8283143,1,2,7,2,08,27.75,1,2,34,6,NaN,NaN,NaN,NaN,NaN,NaN,ATLÁNTICO
2,8283145,1,1,7,2,08,59.11,1,1,45,6,NaN,1.00,1.00,0141,"1,100,000.00",Cría de ganado bovino y bufalino,ATLÁNTICO
3,8283145,1,2,7,2,08,59.11,1,2,50,6,NaN,NaN,NaN,NaN,NaN,NaN,ATLÁNTICO
4,8283145,1,3,7,2,08,59.11,1,1,19,6,NaN,1.00,7.00,0141,"820,000.00",Cría de ganado bovino y bufalino,ATLÁNTICO


2. Renombra las columnas del DataFrame `df_empleo` utilizando el diccionario `mapa_nombres`.

In [55]:
mapa_nombres = {
'DIRECTORIO': 'id_vivienda',
'SECUENCIA_P': 'id_hogar',
'ORDEN': 'id_persona',
'MES': 'mes',
'CLASE': 'clase',
'DPTO': 'departamento',
'FEX_C18': 'factor_expansion',
'PT': 'poblacion_total',
'P3271': 'sexo',
'P6040': 'edad',
'P6080': 'grupo_etnico',
'P3043': 'nivel_educativo',
'OCI': 'ocupados',
'P6430': 'categoria_ocupacional',
'RAMA4D_R4': 'cod_ciiu',
'INGLABO': 'ingreso_laboral',
'DESCRIPCION_CIIU': 'desc_ciiu',
'NOMBRE_DPTO': 'nombre_dpto'}

In [56]:
df_empleo.rename(columns=mapa_nombres, inplace=True)
df_empleo.head()


,id_vivienda,id_hogar,id_persona,mes,clase,departamento,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,ingreso_laboral,desc_ciiu,nombre_dpto
0,8283143,1,1,7,2,08,27.75,1,1,72,6,NaN,1.00,7.00,0143,"790,000.00",Cría de ovejas y cabras,ATLÁNTICO
1,8283143,1,2,7,2,08,27.75,1,2,34,6,NaN,NaN,NaN,NaN,NaN,NaN,ATLÁNTICO
2,8283145,1,1,7,2,08,59.11,1,1,45,6,NaN,1.00,1.00,0141,"1,100,000.00",Cría de ganado bovino y bufalino,ATLÁNTICO
3,8283145,1,2,7,2,08,59.11,1,2,50,6,NaN,NaN,NaN,NaN,NaN,NaN,ATLÁNTICO
4,8283145,1,3,7,2,08,59.11,1,1,19,6,NaN,1.00,7.00,0141,"820,000.00",Cría de ganado bovino y bufalino,ATLÁNTICO


3. Obtenga las dimensiones de la base

In [57]:
df_empleo.shape

(409127, 18)

# Ejercicio 3: Revisión de duplicados

Un paso crucial en la limpieza es asegurar que no haya observaciones duplicadas que puedan sesgar nuestro análisis.

1. Verifica si hay filas duplicadas en `df_empleo`.

In [58]:
df_empleo[df_empleo.duplicated()]

,id_vivienda,id_hogar,id_persona,mes,clase,departamento,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,ingreso_laboral,desc_ciiu,nombre_dpto


2. Cuenta cuántas filas duplicadas existen.

In [59]:
df_empleo.duplicated().sum()


np.int64(0)

3. Elimina las filas duplicadas, si las hay.

4. Verifica si hay filas duplicadas específicamente en las columnas 'id_vivienda', 'id_hogar', 'id_persona'.

In [60]:
df_empleo.duplicated(subset=['id_vivienda','id_hogar','id_persona']).sum()

np.int64(0)

# Ejercicio 4: Valores faltantes

Los datos del mundo real raramente están completos. Aquí, trataremos los valores faltantes en la columna de ingresos.


1. Calcula el número de valores nulos (`NaN`) en la columna `ingreso_laboral` **antes** de cualquier modificación.

In [61]:
df_empleo['ingreso_laboral'].isna().sum()


np.int64(231144)

2. En la GEIH, un ingreso faltante a menudo significa que la persona no reportó ingresos o no aplica (ej. no trabaja). Una estrategia común es imputar estos valores con `0`. Rellena los valores `NaN` en la columna `ingreso_laboral` con `0`.

In [62]:
df_empleo['ingreso_laboral'] = df_empleo['ingreso_laboral'].fillna(0)
df_empleo.tail()

,id_vivienda,id_hogar,id_persona,mes,clase,departamento,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,ingreso_laboral,desc_ciiu,nombre_dpto
409122,8473907,1,1,12,2,73,378.31,1,2,61,6,NaN,1.00,3.00,9700,"1,400,000.00",Actividades de los hogares individuales como e...,TOLIMA
409123,8473908,1,1,12,1,54,534.55,1,1,58,6,9.00,1.00,2.00,8530,"3,000,000.00",Establecimientos que combinan diferentes nivel...,NORTE DE SANTANDER
409124,8473908,1,2,12,1,54,534.55,1,2,51,6,9.00,1.00,2.00,8412,"7,956,000.00",Actividades ejecutivas de la administración pú...,NORTE DE SANTANDER
409125,8473908,1,3,12,1,54,534.55,1,2,31,6,7.00,NaN,NaN,NaN,0.00,NaN,NORTE DE SANTANDER
409126,8473908,1,4,12,1,54,534.55,1,2,26,6,7.00,1.00,4.00,4791,"500,000.00",Comercio al por menor realizado a través de in...,NORTE DE SANTANDER


# Ejercicio 5: Decodificar variables



Las variables categóricas a menudo necesitan ser "traducidas" para hacerlas más entendibles para el análisis.

1. Usa los siguientes diccionarios para reemplazar los códigos numéricos por etiquetas de texto en las siguientes columnas: 'clase', 'sexo', 'grupo_etnico', 'nivel_educativo' y 'categoria_ocupacional'. Muestra los conteos de las nuevas categorías usando `.value_counts()` para verificar el cambio en cada una de las columnas.

In [63]:
dic_clase = {1: 'Cabeceras', 2: 'Centros poblados y rural disperso'}
dic_sexo = {1: 'Hombre', 2: 'Mujer'}
dic_etnico = {1: 'Indígena', 2: 'Gitano (Rom)', 3: 'Raizal del archipiélago', 4: 'Palenquero',
              5: 'Negro, mulato (afrodescendiente)', 6: 'Ninguno de los anteriores'}
dic_nivel_educativo = {1: 'Ninguno', 2: 'Educación media', 3: 'Educación media',
                       4: 'Educación media', 5: 'Técnica profesional', 6: 'Tecnológica',
                       7: 'Universitaria', 8: 'Especialización', 9: 'Maestría',
                       10: 'Doctorado', 99: 'No sabe, no informa'}
dic_cat_ocu = {1: 'Asalariado', 2: 'Asalariado', 3: 'Asalariado',
               8: 'Asalariado', 4: 'Independiente', 5: 'Independiente',
               9: 'Independiente', 6: 'Trabajador sin remuneración',
               7: 'Trabajador sin remuneración'}

In [64]:
df_empleo['clase'] = df_empleo['clase'].map(dic_clase)
df_empleo['clase'].value_counts()


,count
clase,
Cabeceras,356377
Centros poblados y rural disperso,52750


In [65]:
df_empleo['sexo'] = df_empleo['sexo'].map(dic_sexo)
df_empleo['sexo'].value_counts()



,count
sexo,
Mujer,218597
Hombre,190530


In [66]:
df_empleo['grupo_etnico'] = df_empleo['grupo_etnico'].map(dic_etnico)
df_empleo['grupo_etnico'].value_counts()


,count
grupo_etnico,
Ninguno de los anteriores,350562
"Negro, mulato (afrodescendiente)",39032
Indígena,18420
Raizal del archipiélago,925
Palenquero,120
Gitano (Rom),68


In [67]:
df_empleo['nivel_educativo'] = df_empleo['nivel_educativo'].map(dic_nivel_educativo)
df_empleo['nivel_educativo'].value_counts()


,count
nivel_educativo,
Educación media,116635
Universitaria,32517
Técnica profesional,24503
Ninguno,12963
Tecnológica,10640
Especialización,7927
Maestría,3154
Doctorado,339
"No sabe, no informa",1


In [68]:
df_empleo['categoria_ocupacional'] = df_empleo['categoria_ocupacional'].map(dic_cat_ocu)
df_empleo['categoria_ocupacional'].value_counts()

,count
categoria_ocupacional,
Asalariado,92016
Independiente,82498
Trabajador sin remuneración,7339


# Ejercicio 6: Reorganizar columnas

1. Reorganiza las columnas con el siguiente orden: 'id_vivienda', 'id_hogar', 'id_persona', 'mes', 'clase', 'departamento', 'nombre_dpto',     'factor_expansion', 'poblacion_total', 'sexo', 'edad', 'grupo_etnico', 'nivel_educativo', 'ocupados', 'categoria_ocupacional', 'cod_ciiu', 'desc_ciiu', 'ingreso_laboral'

In [69]:
orden_columnas = ['id_vivienda', 'id_hogar', 'id_persona', 'mes', 'clase',
    'departamento', 'nombre_dpto', 'factor_expansion', 'poblacion_total',
    'sexo', 'edad', 'grupo_etnico', 'nivel_educativo', 'ocupados',
    'categoria_ocupacional', 'cod_ciiu', 'desc_ciiu', 'ingreso_laboral']

df_empleo = df_empleo[orden_columnas]
df_empleo.head()

,id_vivienda,id_hogar,id_persona,mes,clase,departamento,nombre_dpto,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,desc_ciiu,ingreso_laboral
0,8283143,1,1,7,Centros poblados y rural disperso,08,ATLÁNTICO,27.75,1,Hombre,72,Ninguno de los anteriores,NaN,1.00,Trabajador sin remuneración,0143,Cría de ovejas y cabras,"790,000.00"
1,8283143,1,2,7,Centros poblados y rural disperso,08,ATLÁNTICO,27.75,1,Mujer,34,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
2,8283145,1,1,7,Centros poblados y rural disperso,08,ATLÁNTICO,59.11,1,Hombre,45,Ninguno de los anteriores,NaN,1.00,Asalariado,0141,Cría de ganado bovino y bufalino,"1,100,000.00"
3,8283145,1,2,7,Centros poblados y rural disperso,08,ATLÁNTICO,59.11,1,Mujer,50,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
4,8283145,1,3,7,Centros poblados y rural disperso,08,ATLÁNTICO,59.11,1,Hombre,19,Ninguno de los anteriores,NaN,1.00,Trabajador sin remuneración,0141,Cría de ganado bovino y bufalino,"820,000.00"


# Ejercicio 7: Cambiar tipo de dato


Para realizar cálculos y análisis correctamente, las columnas deben tener el tipo de dato adecuado (numérico, categórico, etc.).

1.  Muestra en pantalla los tipos de datos actuales de `df_empleo`.

In [70]:
df_empleo.dtypes


,0
id_vivienda,int64
id_hogar,int64
id_persona,int64
mes,int64
clase,object
departamento,object
nombre_dpto,object
factor_expansion,float64
poblacion_total,int64
sexo,object


2.  Las siguientes columnas representan categorías: 'clase', 'departamento', 'nombre_dpto', 'sexo', 'grupo_etnico', 'nivel_educativo', 'categoria_ocupacional', 'cod_ciiu', 'desc_ciiu'. Conviértelas al tipo `category` para optimizar el uso de memoria y facilitar análisis futuros.

In [71]:
columnas_categoricas = ['clase', 'departamento', 'nombre_dpto', 'sexo',
    'grupo_etnico', 'nivel_educativo', 'categoria_ocupacional','cod_ciiu',
      'desc_ciiu']

df_empleo[columnas_categoricas] = df_empleo[columnas_categoricas].astype('category')
df_empleo.dtypes

,0
id_vivienda,int64
id_hogar,int64
id_persona,int64
mes,int64
clase,category
departamento,category
nombre_dpto,category
factor_expansion,float64
poblacion_total,int64
sexo,category


# Ejercicio 8: Ordenar el dataframe (5%)

1. Ordena el dataframe por la columna "edad" de mayor a menor. Observa la columna 'edad' y escribe en una celda de markdown cuál fue la edad mínima y la edad máxima de las personas que respondieron la encuesta de la GEIH.

In [72]:
df_empleo.sort_values(by=['edad'], ascending = False)

,id_vivienda,id_hogar,id_persona,mes,clase,departamento,nombre_dpto,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,desc_ciiu,ingreso_laboral
256896,8395695,1,1,10,Cabeceras,18,CAQUETÁ,40.61,1,Mujer,108,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
5135,8286476,1,5,7,Cabeceras,27,CHOCÓ,31.65,1,Mujer,108,"Negro, mulato (afrodescendiente)",NaN,NaN,NaN,NaN,NaN,0.00
32904,8299104,1,3,7,Centros poblados y rural disperso,13,BOLÍVAR,128.58,1,Mujer,106,"Negro, mulato (afrodescendiente)",NaN,NaN,NaN,NaN,NaN,0.00
256662,8395585,1,3,10,Cabeceras,17,CALDAS,66.42,1,Mujer,106,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
396633,8468208,1,4,12,Centros poblados y rural disperso,25,CUNDINAMARCA,"1,231.96",1,Mujer,106,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
245532,8391286,1,5,10,Cabeceras,63,QUINDÍO,151.42,1,Mujer,0,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
145507,8348957,1,4,9,Cabeceras,50,META,241.71,1,Hombre,0,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
185317,8365558,1,3,9,Cabeceras,85,CASANARE,299.88,1,Mujer,0,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00
366733,8446239,1,6,12,Cabeceras,08,ATLÁNTICO,"1,371.41",1,Hombre,0,Ninguno de los anteriores,NaN,NaN,NaN,NaN,NaN,0.00


In [73]:
edad_min = df_empleo['edad'].min()
edad_max = df_empleo['edad'].max()

print("Edad mínima:", edad_min)
print("Edad máxima:", edad_max)

Edad mínima: 0
Edad máxima: 108


La edad mínima observada en la base de datos es 0 años, mientras que la edad máxima es 108 años.

---




# Ejercicio 9: Groupby

Agrupar los datos por categorías, nos permite realizar análisis más profundos y obtener insights valiosos, que a simple vista no vemos en la base de datos.

1. Crea un nuevo dataframe llamado "df_sexo" donde se agrupe por 'sexo' y se calcule la suma de las columnas 'poblacion_total' y 'ocupados'. Crea una nueva columna que se llame 'porcentaje_ocupados' donde dividas las columna 'ocupados' sobre 'poblacion_total'. Al final usa reset_index() para convertir las categorías de sexo en una columna.

In [74]:
df_sexo = df_empleo.groupby('sexo')[['poblacion_total', 'ocupados']].sum()
df_sexo['porcentaje_ocupados'] = df_sexo['ocupados'] / df_sexo['poblacion_total']
df_sexo = df_sexo.reset_index()
df_sexo

,sexo,poblacion_total,ocupados,porcentaje_ocupados
0,Hombre,190530,"100,790.00",0.53
1,Mujer,218597,"81,063.00",0.37


2. Crea un nuevo dataframe llamado "df_educacion" donde se agrupe por 'nivel_educativo' y se calcule la suma de las columnas 'poblacion_total' y 'ocupados'. Crea una nueva columna que se llame 'porcentaje_ocupados' donde dividas las columna 'ocupados' sobre 'poblacion_total'. Al final usa reset_index() para convertir las categorías de nivel educativo en una columna.

In [75]:
df_educacion = df_empleo.groupby('nivel_educativo')[['poblacion_total', 'ocupados']].sum()
df_educacion['porcentaje_ocupados'] = df_educacion['ocupados'] / df_educacion['poblacion_total']
df_educacion = df_educacion.reset_index()
df_educacion

,nivel_educativo,poblacion_total,ocupados,porcentaje_ocupados
0,Doctorado,339,294.00,0.87
1,Educación media,116635,"66,945.00",0.57
2,Especialización,7927,"6,269.00",0.79
3,Maestría,3154,"2,701.00",0.86
4,Ninguno,12963,"2,539.00",0.20
5,"No sabe, no informa",1,1.00,1.00
6,Tecnológica,10640,"7,928.00",0.75
7,Técnica profesional,24503,"16,989.00",0.69
8,Universitaria,32517,"23,461.00",0.72


3. En una celda de Markdown escribe qué conclusiones puedes sacar de la información obtenida en df_sexo y df_educacion.

En df_sexo se observa que los hombres tienen una mayor tasa de ocupación (53%) en comparación con las mujeres (37%), lo que evidencia una brecha en el acceso al empleo.

En df_educacion, se identifican diferencias importantes entre grupos, donde algunos presentan altas tasas de ocupación (75%) y otros, como la población indígena, niveles más bajos (20%), lo que sugiere desigualdades en el mercado laboral.

# Ejercicio 10: Filtrar datos

1. A partir de df_empleo, crea el DataFrame `df_ing` con observaciones donde `ingreso_laboral` no sea nulo y además sea mayor estrictamente al salario mínimo para 2026 (con subsidio de transporte).

In [76]:
salario_min = 2000000

df_ing = df_empleo[(df_empleo['ingreso_laboral'].notna()) &
    (df_empleo['ingreso_laboral'] > salario_min)]

df_ing.head()

,id_vivienda,id_hogar,id_persona,mes,clase,departamento,nombre_dpto,factor_expansion,poblacion_total,sexo,edad,grupo_etnico,nivel_educativo,ocupados,categoria_ocupacional,cod_ciiu,desc_ciiu,ingreso_laboral
6,8283147,1,1,7,Centros poblados y rural disperso,08,ATLÁNTICO,57.99,1,Hombre,36,Ninguno de los anteriores,NaN,1.00,Asalariado,5022,Transporte fluvial de carga,"2,623,500.00"
41,8283163,1,1,7,Centros poblados y rural disperso,08,ATLÁNTICO,496.47,1,Hombre,39,Ninguno de los anteriores,Tecnológica,1.00,Asalariado,1103,"Producción de malta, elaboración de cervezas y...","3,000,000.00"
111,8283213,1,2,7,Cabeceras,66,RISARALDA,360.58,1,Hombre,24,Ninguno de los anteriores,Universitaria,1.00,Asalariado,8610,"Actividades de hospitales y clínicas, con inte...","2,700,000.00"
204,8283275,1,2,7,Cabeceras,66,RISARALDA,208.52,1,Mujer,35,Ninguno de los anteriores,Técnica profesional,1.00,Asalariado,4631,Comercio al por mayor de productos alimenticios,"2,200,000.00"
205,8283277,1,1,7,Centros poblados y rural disperso,66,RISARALDA,537.44,1,Hombre,69,Ninguno de los anteriores,Educación media,1.00,Asalariado,5611,Expendio a la mesa de comidas preparadas,"2,450,000.00"


2. ¿Cuántas personas tienen un salario estrictamente mayor al salario mínimo?

In [77]:
df_ing.shape[0]

38093

3. Crea el dataframe df_mujeres donde se encuentren solamente mujeres que están ubicadas en el departamento del Tolima, que son asalariadas y cuyo nivel educativo es educación media o superior.

In [86]:
df_mujeres = df_empleo[
    (df_empleo['sexo'] == 'Mujer') &
    (df_empleo['nombre_dpto'] == 'TOLIMA') &
    (df_empleo['categoria_ocupacional'] == 'Asalariado') &
    (df_empleo['nivel_educativo'].isin(['Educación media', 'Técnica profesional','Tecnológica',
        'Universitaria', 'Especialización','Maestría', 'Doctorado']))]

4. ¿Cuáles son las dimensiones de df_mujeres?

In [87]:
df_mujeres.shape



(1137, 18)